python/ribasim/ribasim/utils.py

```diff
@@ -89,8 +89,19 @@ def _add_cf_attributes(ds, timeseries_id):
             "references": "https://ribasim.org",
         }
     )
-    ds["time"].attrs.update({"standard_name": "time", "axis": "T"})
-    ds[timeseries_id].attrs.update({"cf_role": "timeseries_id"})
+    ds["time"].attrs.update({"standard_name": "time", "axis": "T", "long_name": "time"})
+    ds[timeseries_id].attrs.update(
+        {"cf_role": "timeseries_id", "long_name": "station identification code"}
+    )
+    if realization:
+        # https://confluence.ecmwf.int/display/COPSRV/Metadata+recommendations+for+encoding+NetCDF+products+based+on+CF+convention#MetadatarecommendationsforencodingNetCDFproductsbasedonCFconvention-3.4Realizationdiscretecoordinates
```

Comment:
A link on its own is not that useful, could you add a oneline summary? Or just rename `realization` to `discrete_axis` or similar?